### PR TITLE
add fuzzing and invariant testing page

### PR DESCRIPTION
**Pull request by MarcoWorms**

### CODE REVIEW

File: docs/security/fuzzing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Fuzzing and Invariant Testing
```
Comment (storming0x) [inline docs/security/fuzzing.md:1]: Can we change the name of the file and title to Testing? and testing.md

its general information and guidelines for testing smart contract code. Fuzzing is a sub section

**Review by storming0x — COMMENTED** [review]
LGTM, will leave some time for others to review before approving

**Review by storming0x — APPROVED** [review]
LGTM!

**Review by 0xBasically — APPROVED** [review]
lgtm

**Pull request merged**